src/SFML/CMakeLists.txt

```diff
@@ -6,8 +6,14 @@ if(SFML_OS_WINDOWS)
     list(APPEND CMAKE_INCLUDE_PATH "${PROJECT_SOURCE_DIR}/extlibs/headers")
     if(SFML_COMPILER_GCC OR (SFML_COMPILER_CLANG AND MINGW))
         if(ARCH_32BITS)
+            if(SFML_UCRT)
+                message(FATAL_ERROR "UCRT32 environment is not supported")
+            endif()
```

Comment:
I'm of the opinion that we should just do it. If you could build the other libraries, then this is done in a breeze by installing a 32-bits compiler.